pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -646,7 +646,7 @@ func (b *volumeBinder) checkBindings(pod *v1.Pod, bindings []*BindingInfo, claim
 			}
 
 			if err := volumeutil.CheckNodeAffinity(pv, node.Labels); err != nil {
-				return false, fmt.Errorf("pv %q node affinity doesn't match node %q: %v", pv.Name, node.Name, err)
+				return false, fmt.Errorf("pv %q node affinity doesn't match node %q: %w", pv.Name, node.Name, err)
 			}
 		}
 
```

Comment:
(comment not related with this line)

Do you want to cleanup all `%v.*err` occurrences?

```
⇒  ag "%v.*err" pkg/controller/volume/scheduling/scheduler_binder.go
367:		klog.V(5).Infof("AssumePodVolumes: GetBindVolumeToClaim for pod %q, PV %q, PVC %q.  newPV %p, dirty %v, err: %v",
496:			klog.V(4).Infof("updating PersistentVolume[%s]: binding to %q failed: %v", binding.pv.Name, claimKey, err)
552:		klog.V(4).Infof("Could not get a CSINode object for the node %q: %v", node.Name, err)
564:		klog.Errorf("failed to get pod %s/%s from the lister: %v", pod.Namespace, pod.Name, err)
698:		return false, nil, fmt.Errorf("error getting PVC %q: %v", pvcKey, err)
769:		klog.V(4).Infof("Could not get a CSINode object for the node %q: %v", node.Name, err)
786:			klog.V(4).Infof("PersistentVolume %q, Node %q mismatch for Pod %q: %v", pvName, node.Name, podName, err)
964:		klog.Errorf("unexpected error converting %+v to a label selector: %v", capacity.NodeTopology, err)
1037:		return nil, fmt.Errorf("could not get plugin name from pv: %v", err)
1050:		return nil, fmt.Errorf("could not translate pv: %v", err)
```

Reply:
I want to keep this PR as small as possible (although I couldn't contain myself from doing some cleanups) just in case we need to cherry pick. I opened #94695 for the rest.